src/assets/database.sql

```diff
@@ -91,6 +92,16 @@ INSERT INTO `commands` (`id`, `name`, `description`, `examples`, `tool`, `type`,
 (96, 'Source addresses (list separated by +)', NULL, NULL, 16, 'input', '-t source', 'Example: 203.0.113.0/24+198.51.100.0/24', NULL, NULL),
 (97, 'Destination addresses (list separated by +)', NULL, NULL, 16, 'input', '-t destination', 'Example: 192.0.2.0/24+198.18.0.0/15', NULL, NULL);
 
+--
+-- Populate data for `commands`: hping3
+--
+
+INSERT INTO `commands` (`id`, `name`, `description`, `examples`, `tool`, `type`, `command`, `value`, `example`, `sudo`, `category`) VALUES
+(98, 'Packet count', 'Number of packages to send', 'hping3 --count 10', 18, 'input', '--count', '10', '10', NULL, 'COUNTER'),
```

Comment:
can we change `'--count', '10'` with `'--count 10'`?

Reply:
Not in this case because we want the user to be capable of changing it for a lab.